### PR TITLE
[Merged by Bors] - systest: update README and Makefile

### PR DESCRIPTION
**Pull request by countvonzero**
## Motivation

add more instructions for running tests locally

## Changes

- fix Makefile to remove node selector
- update README

## Test Plan

manual testing

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

File: systest/Makefile [hunk 1]
```diff
@@ -7,7 +7,6 @@ clusters ?= 1
 size ?= 10
 level ?= debug
 bootstrap ?= 5m
-node_selector ?= cloud.google.com/gke-nodepool=dmitry-ci-preemtible-1
```
Comment (dshulyak) [inline systest/Makefile:10]: i meant to remove `cloud.google.com/gke-nodepool=dmitry-ci-preemtible-1` but keep everything else. so by default node_selector will be empty, and will be overwritten only when tests are running in the cloud

**Review by dshulyak — APPROVED** [review]
please fix node_selector

File: systest/Makefile (hunk 1 above)
Comment (countvonzero) [reply systest/Makefile:10]: done

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**